external/storm-redis/src/main/java/org/apache/storm/redis/common/mapper/RedisDataTypeDescription.java

```diff
@@ -23,7 +23,7 @@
  * RedisDataTypeDescription defines data type and additional key if needed for lookup / store tuples.
  */
 public class RedisDataTypeDescription implements Serializable {
-    public enum RedisDataType { STRING, HASH, LIST, SET, SORTED_SET, HYPER_LOG_LOG, GEO }
+    public enum RedisDataType { STRING, HASH, LIST, SET, SORTED_SET, HYPER_LOG_LOG, GEO, EXISTS }
```

Comment:
`RedisMapper` point out `RedisDataType` is hard to internal it into storm code ? I'm not sure ...